Dockerfile

```
@@ -8,6 +8,7 @@ RUN apt-get update && \
     apt-get install -y \
         zsh \
         ros-dev-tools && \
+        ros-humble-ros-gz && \
```

Please check if this works. To my best knowledge it shouldn't. The fix looks like this:
```suggestion
        ros-dev-tools \
        ros-humble-ros-gz && \
```